docs/en/engines/database-engines/materialize-mysql.md

```diff
@@ -0,0 +1,160 @@
+---
+toc_priority: 29
+toc_title: MaterializeMySQL
+---
+
+# MaterializeMySQL {#materialize-mysql}
+
+ Creates the tables, existing in MySQL, in the ClickHouse and inputs the data at the same time. 
```

Comment:
1. Creates database with all the tables existing in MySQL?
2. What does it mean "at the same time"?